openfn/package-metadata.json

```
@@ -0,0 +1,41 @@
+{
+  "id": "openfn",
+  "name": "OpenFn",
+  "description": "Lightning ⚡ (OpenFn v2) is a workflow automation platform that&#39;s used to automate critical business processes and integrate information systems.",
+  "type": "infrastructure",
+  "version": "0.0.1",
+  "dependencies": ["database-postgres"],
+  "environmentVariables": {
+    "DATABASE_URL": "postgresql://openfn:instant101@postgres-1:5432/lightning_dev",
```

```suggestion
    "OPENFN_DATABASE_URL": "postgresql://openfn:instant101@postgres-1:5432/lightning_dev",
```
I suggest we have all variables pre-appended with **OPENFN** for easier management of all platform variables in a single environment file.